src/docs/self-hosted/geolocation.mdx

```diff
@@ -16,6 +16,8 @@ With this configuration file in place, subsequent runs of Sentry's `install.sh`
 
 1. For the `web` service: **User Settings > Security > Session History** should display country code and region (for example, "US (CA)") underneath the IP addresses in the table.
 
+Note that it's normal to see the `sentry_onpremise_geoipupdate_1` container exit soon after startup, since updating the geolocation database is a one-off batch process and not a long-running job.
```

Comment:
I'd put this into an `<Alert>` box like in https://github.com/getsentry/develop/blob/a140655c64c38135caff40aa677452364c2007b2/src/docs/self-hosted/index.mdx#L31-L33